usage_examples/multitask_openai/openai_multitask_fewshot.cfg

```diff
@@ -0,0 +1,41 @@
+[paths]
+examples = null
+
+[nlp]
+lang = "en"
+pipeline = ["llm_spancat", "llm_textcat"]
+
+[components]
+
+[components.llm_spancat]
+factory = "llm"
+
+[components.llm_spancat.task]
```

Comment:
Why not `llm_ner`? I mean, not that it wouldn't work, but it seems confusing for an example?

Reply:
Ah right, I made this with a `SpanCat` in mind to add an example, but changed course to avoid waiting for the other PR. Thanks, I'll change that.